docs/source/examples/substrafl/go_further/run_diabetes_substrafl.ipynb

```diff
@@ -0,0 +1,698 @@
+{
```

Comment:
<p><code>Compute Plan Builder</code> and <code>Decorator</code> are not displaying correctly, for some reasons</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-documentation/pull/368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>